org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsd/participants/XSDErrorCode.java

```diff
@@ -130,6 +134,7 @@ public static Range toLSPRange(XMLLocator location, XSDErrorCode code, Object[]
 		case src_element_2_1:
 		case src_element_3:
 		case src_import_1_2:
+		case src_import_3_1:

```

Comment:
I think for src_import_3_1 it should be better to highlight the namespace attribute.